ppafm/chemistry.py

```diff
@@ -73,15 +73,12 @@ def findTris(bonds, neighs):
         for i in a_ngs:
             if i in b_ngs:
                 common.append(i)
-        # print "bond ",b," common ",common
         ncm = len(common)
         if ncm > 2:
-            if iDebug > 0:
-                print("WARRNING: bond ", b, " common neighbors ", common)
+            logger.debug(f"WARNING: bond {b} common neighbors {common}")
```

Comment:
I know, that even before it was in the `debug` mode, but should not **warnings** be part of `info`?

Reply:
Changed to `warning`.